File: app/src/main/java/com/mapmyindia/sdk/demo/java/activity/DrivingRangePluginActivity.java
package com.mapmyindia.sdk.demo.java.activity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;
import androidx.databinding.DataBindingUtil;

import com.mapmyindia.sdk.demo.R;
import com.mapmyindia.sdk.demo.databinding.ActivityDrivingRangePluginBinding;
import com.mapmyindia.sdk.demo.java.settingscreen.DrivingRangeSettingActivity;

public class DrivingRangePluginActivity extends AppCompatActivity {

    private ActivityDrivingRangePluginBinding mBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mBinding = DataBindingUtil.setContentView(this, R.layout.activity_driving_range_plugin);
        mBinding.btnOpenDrivingRange.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(DrivingRangePluginActivity.this, DrivingRangeActivity.class));
            }
        });
        mBinding.btnSetting.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(DrivingRangePluginActivity.this, DrivingRangeSettingActivity.class));
            }
        });
    }
}

File: app/src/main/java/com/mapmyindia/sdk/demo/kotlin/activity/DrivingRangePluginActivity.kt
package com.mapmyindia.sdk.demo.kotlin.activity

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.mapmyindia.sdk.demo.R
import com.mapmyindia.sdk.demo.databinding.ActivityDrivingRangePluginBinding
import com.mapmyindia.sdk.demo.kotlin.settingscreen.DrivingRangeSettingActivity

class DrivingRangePluginActivity : AppCompatActivity() {

    private lateinit var mBinding: ActivityDrivingRangePluginBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        mBinding = DataBindingUtil.setContentView(this, R.layout.activity_driving_range_plugin)
        mBinding.btnOpenDrivingRange.setOnClickListener {
            startActivity(Intent(this, DrivingRangeActivity::class.java))
        }
        mBinding.btnSetting.setOnClickListener {
            startActivity(Intent(this, DrivingRangeSettingActivity::class.java))
        }
    }
}